cds_migrator_kit/videos/weblecture_migration/transform/folders_transform.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-2025 CERN.
+#
+# CDS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS transform step module."""
+import logging
+from copy import deepcopy
+from pathlib import Path
+
+import requests
+from invenio_rdm_migrator.streams.records.transform import RDMRecordTransform
+
+from cds_migrator_kit.errors import (
+    ManualImportRequired,
+    MissingRequiredField,
+    UnexpectedValue,
+)
+from cds_migrator_kit.transform.dumper import CDSRecordDump
+
+cli_logger = logging.getLogger("migrator")
+
+
+class FoldersTransform(RDMRecordTransform):
```

Comment:
```suggestion
class GenerateFilesFoldersTransform(RDMRecordTransform):
```
maybe just rename this